lib/plenario/schemas/data_set_field.ex

```diff
@@ -23,6 +23,7 @@ defmodule Plenario.Schemas.DataSetField do
     JSON: "jsonb"
   ]
 
+  @derive {Poison.Encoder, only: [:id, :name, :type, :description]}
```

Comment:
I don't think including `id`s is useful -- we're not exposing endpoints for fields or constraints or any of the other sub-elements of `Meta`.